testing/testcafe/tests/editors/radioGroup/validationMessage.ts

```diff
@@ -0,0 +1,55 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import url from '../../../helpers/getPageUrl';
+import CheckBox from '../../../model/checkBox';
+import createWidget from '../../../helpers/createWidget';
+
+fixture`Radio Group ValidationMessage`
+  .page(url(__dirname, '../pages/t1020449.html'));
```

Comment:
why we don't use `container.html` here?

Reply:
We may use additional html pages to test scenarios that difficult emulate in test body. Only 2 such test pages left: dateBoxGeometry.html & t901779.html.